### DISCUSSION TITLE
How make portable version smplayer default player

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#967] [General]
I downloaded the portable smplayer and extracted it but when I go to win10 file association I'm not sure what I should click. Thanks.

**Comment by Raza-0007** [comment]
`Right click` on the file you want to play, then choose `open with` and if `smplayer (portable)` is not listed, select `choose another app`, navigate to the location of portable `smplayer.exe` and click to `Always` open this file type with `smplayer.exe`. Repeat for all file extensions.

You can also set smplayer portable as your default media player app from `Windows - > Settings -> Apps - > Default apps`

~Raza
